### ISSUE TITLE
A11y_Visual Studio Code_Customizations and Adding Extensions_Resize: Some data in the Extension page is not visible when Resize is applied

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122906]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#Resize;#WCAG1.4.4;#Win32;#Zoom;#AINotInScope;#Win10;VisualStudioCode-Web-Apr2021;

### Environment Details:​
Application Name: Visual Studio Code
Standard Violated:WCAG1.4.4

### Repro Steps:​
1. Change the screen display resolution to an "average" mid-range resolution (1280*768 or near to it 1280x800)
2. Set the slider to 200% under 'Make text bigger' in Display settings in Ease of Access.
3. Open Visual Studio Code editor
4. Navigate to the extensions in the sidebar options using "Ctrl+Shift+X" key
5. Tab till any extension under popular (ex. python extension) and press enter
6. Verify whether the entire data under the opened extension is visible or not (Complete data in the top section is visible or not)

### Actual Result:
Some data in the Extension page is not visible when Resize is applied

### Expected Result: ​
Entire data in the Extension page should be clearly visible when Resize is applied also.

### User Impact:
Users with low vision who zoom the page for better visibility will face difficulty while navigation if any data is missing in the page.

### Attachment:
[A11y_Visual Studio Code_Customizations and Adding Extensions_Resize Some data in the Extension page is not visible when Resize is applied.zip](https://github.com/microsoft/vscode/files/6419642/A11y_Visual.Studio.Code_Customizations.and.Adding.Extensions_Resize.Some.data.in.the.Extension.page.is.not.visible.when.Resize.is.applied.zip)

**Comment by sandy081** [comment]
@isidorn Assigning to you to get insights

**Comment by isidorn** [comment]
We are in the process of getting an exception for the Reflow
Closing as dup https://github.com/microsoft/vscode/issues/115878
@sana-ajani might know more on the process of getting that exception

**Comment by InduPriya1805** [comment]
GitHubTags:#VisualStudioCode-Win32-Apr2021;
